CP01CNSM/CP01CNSM_R00011_ingest.csv

```diff
@@ -10,20 +10,20 @@ mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP01CNSM
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl36/PHSEN*/SAMI_*.txt,CP01CNSM-MFD35-06-PHSEND000,recovered_inst,Expected,
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl37/syslog/*.syslog.log,CP01CNSM-MFD37-00-DCLENG000,recovered_host,Expected,
 mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl37/optaa2/*.optaa2.log,CP01CNSM-MFD37-01-OPTAAD000,recovered_host,Expected,
 mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl37/ctdbp2/*ctdbp2.log,CP01CNSM-MFD37-03-CTDBPD000,recovered_host,Expected,
 mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl37/CTDBP*/SBE*.hex,CP01CNSM-MFD37-03-CTDBPD000,recovered_inst,Expected,
 mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl37/dosta2/*dosta2.log,CP01CNSM-MFD37-04-DOSTAD000,recovered_host,Expected,
 mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl37/zplsc/*.zplsc.log,CP01CNSM-MFD37-07-ZPLSCC000,recovered_host,Expected,
-mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl37/ZPLSC*/*/*.01A,CP01CNSM-MFD37-07-ZPLSCC000,recovered_inst,Expected,
+#mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl37/ZPLSC*/*/*/*.01A,CP01CNSM-MFD37-07-ZPLSCC000,recovered_inst,Expected,Test parser
 mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/cpm2/syslog/cpm_status.txt,CP01CNSM-RIC21-00-CPMENG000,recovered_host,Expected,
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl26/syslog/*.syslog.log,CP01CNSM-RID26-00-DCLENG000,recovered_host,Expected,
 mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl26/velpt1/*.velpt1.log,CP01CNSM-RID26-04-VELPTA000,recovered_host,Expected,
 mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl26/VELPT_snAQD-*/*.aqd,CP01CNSM-RID26-04-VELPTA000,recovered_inst,Expected,
 mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl26/phsen1/*.phsen1.log,CP01CNSM-RID26-06-PHSEND000,recovered_host,Expected,
-mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl26/PHSEN*/SAMI_P0095_*.txt ,CP01CNSM-RID26-06-PHSEND000,recovered_inst,Expected,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl26/PHSEN*/SAMI_*.txt,CP01CNSM-RID26-06-PHSEND000,recovered_inst,Expected,

```

Comment:
Filemask was not finding data, I removed the space at the end and made it more generic